src/runner/PlutoRunner.jl

```diff
@@ -1610,6 +1612,16 @@ function transform_bond_value(s::Symbol, value_from_js)
     end
 end
 
+function possible_bond_values(s::Symbol)
+    element = get(registered_bond_elements, s, nothing)
+    return try
+        possible_bond_values_ref[](element)
+    catch e
+        @error "Could not collect possible bond values for $s" exception=(e, catch_backtrace())
```

Comment:
Why the try catch rethrow?

Reply:
i think i tried to much to make it look like `transform_bond_value`, this does not seem necessary

Reply:
hmm this can actually be helpful to get the name of the symbol s that failed when developping ?